doc/rados/configuration/mon-lookup-dns.rst

```diff
@@ -21,6 +21,11 @@ which is configured by the *mon_dns_srv_name* configuration directive.
 
 .. confval:: mon_dns_srv_name
 
+.. note:: It is possible to manually designate the search domain by passing the
+   search domain's name followed by an underscore to ``mon_dns_srv_name``. For
+   example: pass ``ceph-mon_example.com`` and make sure that records are placed
+   in ``_ceph-mon._example.com``.
```

Comment:
Followed by an underscore but the example has it preceded by an underscore?

Reply:
Good catch. When I am done eating, I will repair this and credit you as co-author.

Reply:
The example is correct, my wording in the email was slightly misleading. You pass the name of the record  (in this case ceph-mon) and then an underscore, and then the search domain.

Reply:
Roger that. I'll fix it up soon.

Reply:
@LanderDuncan: It's the end of my day, and I am too tired and dumb to know exactly which string I should add to the docs here. Let me know what should go here and I'll do it first thing after a nap.

Reply:
note: Instead of using a DNS search domain, you can pass it as a `mon_dns_srv_name` argument. The syntax for this is `<service-name>_<upper-level-domain>`. For example, passing `ceph-mon_example.com` will direct Ceph to look for the SRV record at `_ceph-mon._tcp.example.com`

Reply:
@LanderDuncan, thanks for that. I've made the update.